app/src/main/java/com/github/se/polyfit/ui/screen/PostInfoScreen.kt

```diff
@@ -0,0 +1,170 @@
+package com.github.se.polyfit.ui.screen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.core.content.ContextCompat
+import com.github.se.polyfit.R
+import com.github.se.polyfit.model.nutritionalInformation.Nutrient
+import com.github.se.polyfit.model.post.Post
+
+@Composable
+fun PostInfoScreen(posts: List<Post>, index: Int = 0) {
+
+  if (posts.isEmpty()) {
+    NoPost()
+    return
+  }
+
+  LazyColumn(
+      state = rememberLazyListState(index),
+  ) {
+    items(posts) { post -> PostCard(post = post) }
+  }
+}
+
+@Composable
+private fun PostCard(post: Post) {
+  val modifierPostCard =
+      Modifier.fillMaxWidth()
+          .width(330.dp)
```

Comment:
This is redundant as you set the width twice. You would use WidthIn to set a range.